__tests__/app.test.js

```diff
@@ -273,3 +273,40 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: Deletes the specified comment and responds with no content", () => {
+    const comment_id = 1;
+    return request(app)
+      .delete(`/api/comments/${comment_id}`)
+      .expect(204)
+      .then(() => {
+        const sqlQuery = `SELECT * FROM comments WHERE comment_id = $1`;
+        const queryValues = [comment_id];
+        return db.query(sqlQuery, queryValues).then(({ rows }) => {
+          expect(rows).toHaveLength(0);
+        });
+      });
+  });
```

Comment:
Nice test, and seeing the query here is interesting, but it isnt really needed. This is because this test along with your error handling should give you the confidence that this works as expected and does actually interact with the database